src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { MessageDescriptor } from 'react-intl';
+
+import Icon from '~core/Icon';
+import Heading from '~core/Heading';
+import ProgressBar from '~core/ProgressBar';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import { Address } from '~types/index';
+import { getMainClasses } from '~utils/css';
+import useAvatarDisplayCounter from '~utils/hooks/useAvatarDisplayCounter';
+
+import styles from './VoteResultsWidget.css';
+
+interface Appearance {
+  theme?: 'approve' | 'disapprove';
+}
+
+interface Props {
+  appearance?: Appearance;
+  value: number;
+  maxValue: number;
+  maxPercentage?: number;
+  title: MessageDescriptor;
+  voters?: Address[];
+  maxAvatars?: number;
+}
+
+const displayName = 'VoteResultsWidget';
+
+const VoteResultsWidget = ({
+  appearance = { theme: 'approve' },
+  value,
+  maxValue,
+  maxPercentage = 100,
+  title,
+  voters = [],
+  maxAvatars = 3,
+}: Props) => {
+  const {
+    avatarsDisplaySplitRules,
+    remainingAvatarsCount,
+  } = useAvatarDisplayCounter(maxAvatars, voters);
+  const UserAvatar = HookedUserAvatar({ fetchUser: true });
+  const iconName =
+    appearance.theme === 'approve' ? 'circle-thumbs-up' : 'circle-thumbs-down';
+  const votePercentage = (value * maxPercentage) / maxValue;
+
+  return (
+    <div className={`${styles.wrapper} ${getMainClasses(appearance, styles)}`}>
+      <div className={styles.voteInfoContainer}>
+        <Icon name={iconName} title={title} appearance={{ size: 'medium' }} />
+        <div className={styles.voteResults}>
+          <div className={styles.voteHeading}>
+            <Heading
+              appearance={{
+                theme: 'dark',
+                size: 'small',
+                weight: 'bold',
+                margin: 'none',
+              }}
+              text={title}
+            />
+            <span className={styles.votePercentage}>{votePercentage}%</span>
+          </div>
+          <ProgressBar
+            value={votePercentage}
+            max={maxPercentage}
+            appearance={{ size: 'small', theme: 'transparent' }}
+          />
```

Comment:
If the `VotingResultWidget` theme is `disapprove` the progress bar should have the `danger` theme.

This is the spec:

![Screenshot from 2021-03-15 22-29-45](https://user-images.githubusercontent.com/1193222/111218476-70b45600-85df-11eb-8d1b-8aa86d5c608c.png)

This is how it looks currently _(only the icon changes)_

![Screenshot from 2021-03-15 22-31-35](https://user-images.githubusercontent.com/1193222/111218504-7c078180-85df-11eb-86a8-ea0291439d6e.png)